server/models/Profile.js

```diff
@@ -34,6 +34,21 @@ const profileSchema = new mongoose.Schema({
     type: String,
     default: "",
   },
+  tagLine: {
+    type: String,
+    default: ''
+  },
+  price: {
+    type: String,
+  },
+  isSitter: {
```

Comment:
We don't need this field as we have an `accountType` field which depicts the type of profile